guides/user/kubernetes-v2/automated-rollbacks/index.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: single
+title:  "Configure Automated Rollbacks in the Kubernetes Provider V2"
+sidebar:
+  nav: guides
+---
+
+{% include alpha version="1.6" %}
+
+{% include toc %}
+
+There are clear benefits to having your stored manifests (in Git, GCS, etc...)
+match as closely as possible to what's running in your cluster: you have
+auditability, versioned changes, and a high confidence in what code you have
+running. However, in some cases, pushing changes from someplace like GitHub
+through your build system all the way production can simply take too long, and
+an escape hatch is needed when something has gone seriously wrong.
+
+When using a Deployment object, you can see the history of rollouts in the
+"Clusters" tab:
+
+{% include
+   figure
+   image_path="./revisions.png"
+   caption="Version 4 is active, and version 3 has no pods running."
+%}
+
+Spinnaker exposes "Undo Rollout" functionality in two places, in the [Clusters
+tab](#ad-hoc-rollbacks), and as a [pipeline stage](#automated-rollbacks).
+
+## Ad-hoc Rollbacks
+
+In cases where you see something is immediately wrong and isolated to a
+resource in the "Clusters" tab, you can select "Undo Rollout" from the
+"Actions" dropdown:
+
+{% include
+   figure
+   image_path="./dropdown.png"
+%}
+
+And select the healthy revision to make active again:
+
+{% include
+   figure
+   image_path="./adhoc.png"
+%}
+
+Notice that the old configuration (version 3) will be rolled forward into a
+new version (version 5):
+
+{% include
+   figure
+   image_path="./v005.png"
+%}
+
+## Automated Rollbacks
+
+You can also configure automated rollbacks inside of Spinnaker pipelines. These
+stages are best configured when other stages or branches fail, indicating that
+something has gone wrong in your rollout.
```

Comment:
So an upstream stage fails, and that failure triggers an auto-rollback stage?

Reply:
Yup! I think a guide for configuring stages that run on "failure" merits another guide.

Reply:
"These stages are best configured to run when other stages..."

Reply:
https://github.com/spinnaker/spinnaker.github.io/issues/598